src/govuk/components/character-count/character-count.js

```diff
@@ -153,6 +166,31 @@ CharacterCount.prototype.updateCountMessage = function () {
   }
 
   // Update message
+  countMessage.innerHTML = this.formatUpdateMessage()
+}
+
+// Update screen reader-specific counter
+CharacterCount.prototype.updateScreenReaderCountMessage = function () {
+  var countMessage = this.$screenReaderCountMessage
+
+  // If other the threshold, remove the aria-hidden attribute, allowing screen
+  // readers to announce the content of the element.
+  if (this.isOverThreshold()) {
+    countMessage.removeAttribute('aria-hidden')
+  } else {
+    countMessage.setAttribute('aria-hidden', true)
+  }
+
+  // Update message
+  countMessage.innerHTML = this.formatUpdateMessage()
+}
+
+// Format update message
+CharacterCount.prototype.formatUpdateMessage = function () {
+  var countElement = this.$textarea
```

Comment:
Looks like this is something that's been around for a while and i wonder if this is a good time to update the name of this variable - not sure why it needs to be referred to as 'countElement' when we refer to it elsewhere as 'textArea'?  Any thoughts?

Reply:
Agree. I also noticed that some variables referencing elements are prefixed with `$` while others aren't, so I did a general whip around to try and make the formatting more consistent.